subsys/bluetooth/host/hci_core.h

```diff
@@ -180,8 +180,12 @@ struct bt_dev_le {
 
 #if defined(CONFIG_BT_CONN)
 	/* Controller buffer information */
-	uint16_t			mtu;
+	uint16_t		mtu;
 	struct k_sem		pkts;
+	uint16_t		acl_mtu;
+	struct k_sem		acl_pkts;
+	uint16_t		iso_mtu;
```

Comment:
ifdef iso_mtu and iso_pkts?

Reply:
I guess you are suggesting to make iso related parameters conditional to CONFIG_BT_ISO? Or is it something else?

Reply:
I'm suggesting to wrap iso_mtu and iso_pkts under `#if defined(CONFIG_BT_ISO)` in this struct.